iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -826,6 +856,15 @@ static SmallVector<Value, 4> getAliasingBuffersForResults(
           [&](auto singleResultOp) -> SmallVector<Value, 4> {
             return {getAliasingBufferForResult(b, singleResultOp, bvm)};
           })
+      .Case<scf::ForOp>([&](auto scfFor) -> SmallVector<Value> {
```

Comment:
Nit: Could we move the body into a method. The `Type` -> `Case` is hard to read if there is inlined code for the `Case`.